tests/unit/utils/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""fixtures for share tests"""
+
+import os
+import sys
+
+import pytest
+
+from ansible_navigator.utils.key_value_store import KeyValueStore
+
+
+@pytest.fixture
+def empty_kvs(tmp_path):
```

Comment:
We haven't talked about it, how do you feel about typing in tests?  I've started to add a few and find it useful during debugging and for IDE mouseover docs